junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/config/DisplayNameGeneratorClassParameterConverter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.engine.config;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.platform.commons.logging.Logger;
+import org.junit.platform.commons.logging.LoggerFactory;
+import org.junit.platform.engine.ConfigurationParameters;
+
+/**
+ * @since 5.5
+ */
+class DisplayNameGeneratorClassParameterConverter {
+
+	private static final Logger logger = LoggerFactory.getLogger(DisplayNameGeneratorClassParameterConverter.class);
+
+	Optional<Class<? extends DisplayNameGenerator>> get(ConfigurationParameters configurationParameters, String key) {
+		return configurationParameters.get(key).map(String::trim).map(this::getClassFrom);
+	}
+
+	@SuppressWarnings({ "unchecked" })
+	private Class<? extends DisplayNameGenerator> getClassFrom(String className) {
+		try {
+			Class<?> aClass = Class.forName(className);
```

Comment:
We do not use `Class.forName()`.

Please use `org.junit.platform.commons.util.ReflectionUtils.tryToLoadClass(String)` instead.

Also, please rename `aClass` to `clazz`.

Reply:
acknowledged